ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -127,6 +127,10 @@ objects:
         mkdir -p /go/src/github.com/openshift/
         cp -r /tmp/shared/openshift-azure /go/src/github.com/openshift/
         cd /go/src/github.com/openshift/openshift-azure/
+        mkdir -p /go/src/github.com/openshift/openshift-azure/secrets
```

Comment:
Another approach would be to make `artifactDir` configurable instead of hardcoding to `secrets/`, add it in TestConfig, and set it here instead of doing all of this copying.

Reply:
This is what I was doing with `SECRETSDIR` variable which we removed :D